install_files/ansible-base/roles/common/tasks/create_users.yml

```diff
@@ -41,8 +44,8 @@
 
 - name: add line to each users bashrc to source the SecureDrop additions
   lineinfile:
-    dest: "/home/{{ item }}/.bashrc"
-    line: ". /etc/bashrc.securedrop_additions"
+    dest: /home/{{ item }}/.bashrc
+    line: '. /etc/bashrc.securedrop_additions'
```

Comment:
No, the quotes are not strictly necessary here. I preserved them because they make the use of `.` (rather than `source`) harder to overlook. Choosing between single and double quotes, single makes more sense to me, based on the rule of thumb that double quotes imply variable interpolation. 

A better solution would be to drop the additions file into `/etc/profile.d` and forgo modifying `.bashrc` files altogether. I've added a TODO comment saying as much in 213d4a1.